ksa_compliance/ksa_compliance/doctype/zatca_business_settings/zatca_business_settings.py

```diff
@@ -263,6 +276,52 @@ def on_trash(self) -> None:
         frappe.throw(msg=_("You cannot Delete a configured ZATCA Business Settings"),
                      title=_("This Action Is Not Allowed"))
 
+    def create_tax_account(self) -> str:
+        account_doc = frappe.new_doc("Account")
```

Comment:
Use `cast` to mark the type as `Account` so that you can get code completion and static analysis on `account_doc` usage. This applies to other instances of `new_doc` added to the file when creating linked doctypes.